core/store/src/migrations.rs

```diff
@@ -345,3 +345,15 @@ pub fn migrate_38_to_39(store: &Store) -> anyhow::Result<()> {
     update.commit()?;
     Ok(())
 }
+
+/// Migrates the database from version 39 to 40.
+///
+/// This involves deleting contents of ReceiptIdToShardId column which is now
+/// deprecated and no longer used.
+pub fn migrate_39_to_40(store: &Store) -> anyhow::Result<()> {
```

Comment:
I'm curious how big is this column and how long does it typically take to delete it on a mainnet rpc node?

Reply:
> I'm curious how big is this column and how long does it typically take to delete it on a mainnet rpc node?

Based on this dashboard: it is ~4GB, not sure how long it will take to delete it. I will add some code to time the migration.

Reply:
Oh forgot to add the link, here it is:https://grafana.nearone.org/d/bdifdh18i1hc0e/near-database-insights?orgId=1&viewPanel=1